<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_articulo</class>
 <widget class="QWidget" name="search_articulo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SearchForm</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="l_descripcion">
         <property name="text">
          <string>Descripcion</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="LineEdit" name="le_descripcion"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="l_fk_marca">
         <property name="text">
          <string>Marca</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="LineEdit" name="le_fk_marca"/>
       </item>
       <item row="0" column="5">
        <widget class="LineEdit" name="le_fk_medida"/>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="l_fk_medida">
         <property name="text">
          <string>Medida</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
